From 0adae3b16cef5652455079ffa1814107ebda202b Mon Sep 17 00:00:00 2001
From: root <root@L-DEV-W003.lab.mtl.com>
Date: Mon, 26 Sep 2011 17:51:31 +0300
Subject: [PATCH] srp: fixed compilation warnings

Fix the following compilation warning:
drivers/infiniband/ulp/srp/ib_srp.c: In function 'srp_parse_options':
drivers/infiniband/ulp/srp/ib_srp.c:1854: warning: passing argument 2 of 'match_token' discards qualifiers from pointer target type
---
 drivers/infiniband/ulp/srp/ib_srp.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/infiniband/ulp/srp/ib_srp.c b/drivers/infiniband/ulp/srp/ib_srp.c
index c67c16e..0e8144d 100644
--- a/drivers/infiniband/ulp/srp/ib_srp.c
+++ b/drivers/infiniband/ulp/srp/ib_srp.c
@@ -1804,7 +1804,7 @@ enum {
 				   SRP_OPT_SERVICE_ID),
 };
 
-static const match_table_t srp_opt_tokens = {
+static match_table_t srp_opt_tokens = {
 	{ SRP_OPT_ID_EXT,		"id_ext=%s" 		},
 	{ SRP_OPT_IOC_GUID,		"ioc_guid=%s" 		},
 	{ SRP_OPT_DGID,			"dgid=%s" 		},
-- 
1.7.4.1

